utils/console_menu.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env micropython
+from time import sleep
+from sys import stderr
```

Comment:
What's the expected way people will use this -- is it an example template to start from, or should people be importing it? It seems like the former, given the main function and template code; perhaps we should move it to the demo repo. But I'm open to discussion.

Reply:
Yes. It's an example utility to demonstrate a way to create a 5-button menu sequencer--which will be in high demand for _FIRST_ LEGO League teams; as they are confined to 2 and a half minutes to switch quickly between "mission" routines... The hunt-and-peck of EV3-G and Brickman significantly impact points-per-second metric... so a "menu" is essential--especially with the long launch times of python and (lesser) micropython programs.

Reply:
It would be cool to craft an "advanced" module for these types of classes that reference multiple sensors, other classes, etc. I suspect that folks will indeed compose their own "Robot()" class that closely reflects their specific robot (i.e. my EV3 robot from last year has large motors in B and C, medium motors in A and D, Gyro in 1, Color Sensors in 2 and 3, and a button in 4.) I envision teams implementing their own custom logic in their Robot() class that integrates abstract and advanced capabilities (as follow-line right but stop when left color sensor sees black; or go straight for 2.5 rotations, but ramp up speed, and ramp down near the end to avoid having the robot 'jerk'.)

Reply:
What I was asking was more about this particular PR: is this code really appropriate for `utils`, or should it instead go somewhere else, given that it isn't intended to be run standalone/unmodified?

I get the use case for FLL -- I previously was on an FLL team and  recognize the need. I'm just trying to figure out the most appropriate code organization to not be confusing.